lib/.eslintrc.yaml

```diff
@@ -29,6 +29,8 @@ rules:
       message: "Use `const { Reflect } = primordials;` instead of the global."
     - name: Symbol
       message: "Use `const { Symbol } = primordials;` instead of the global."
+    - name: Set
```

Comment:
Since this now needs a rebase anyway, I'll point out that the entries are in alphabetical order so "Set" should go before "Symbol" and not after it. Can you make that change while rebasing?

Reply:
(Or maybe I"ll make that change while landing....)